docs/ami_lod_rec.md

```diff
@@ -0,0 +1,204 @@
+
+Your source spreadsheet will also need to contain at least one column containing terms/names (values) you want to reconcile against an LoD Authority Source. Multiple values should be separated by '|@|'.
+
+## Step 1: Select the AMI Set you will be working with.
+
+From the main AMI Sets List page, click on your AMI Set's Name, or select the 'Edit' option from the Operations menu on the right-hand side of the Sets list.
+
+![AMI Sets List](images/ami/AMIsetsList.jpg)
+
+## Step 2: Reconcile LoD Tab
+
+Navigate to the Reconcile LoD tab.
+
+![Reconcile LoD Tab](images/ami/AMIreconcileLodTab.jpg)
+
+## Step 3: LoD Reconciling Selections
+
+From the list of columns from your spreadsheet source, select which columns you want to reconcile against LoD providers. 
+
+Under the LoD Sources section, select how your chosen Columns will be LoD reconciled.
+- LoD reconcile options will be on the left, LoD Authority Sources will be on the right.
+- Example: 'local_subjects' will be mapped to 'LoC subjects (LCSH)'
+	
+![AMI LoD Sources](images/ami/AMI_LodSources.jpg)
+	
+??? info "Full list of potential LoD Authority Sources"
+
+ - LoC subjects(LCSH)
+ - LoC Name Authority File (LCNAF)
+ - LoC Genre/Form Terms (LCGFT)
+ - LoC Thesaurus of Graphic Materials (TGN)
+ - LoC MARC List for Geographic Areas
+ - LoC Relators Vocabulary (Roles)
+ - LoC MADS RDF by type:
+	- Corporate Name
+	- Personal Name
+	- Family Name
+	- Topic
+	- Genre Form
+	- Geographic
+	- Temporal
+	- Extraterrestrial Area
+ - VIAF
+ - Getty aat Fuzzy / Terms / Exact Label Match
+ - Wikidata Q Items
+
+To preview the values contained in the column(s) you selected, click the 'Inspect cleaned/split up column values' button.
+	
+![AMI LoD](images/ami/AMIreconcileValuePreview.jpg)
+
+When you are ready, select the 'Process LoD from Source' button.
+
+*Important note: if you have previously run LoD Reconciliation for your AMI set, this action will overwrite any manually corrected LoD on your Processed CSV. Please make sure you have a backup if unsure.*
+
+Depending on the size of your AMI Set, the Reconciliation processing may take a few minutes. 
+	
+![AMI LoD Reconciling Success](images/ami/AMIbatchReconcile.jpg)
+
+When the process is finished, you will see a brief confirmation message.
+	
+![AMI LoD Reconciling Success](images/ami/AMI_LodRecSuccess.jpg)
+
+## Step 4: Edit Reconciled LoD
+
+Open the 'Edit Reconciled LoD' tab.
+
+You will see a table (form) containing:
+- Your Original term values (labels)
+- The CSV Column Header/Key from the source spreadsheet where the value is found
+- A Checked option you can use to denote that an LoD mapping has been reviewed/revisioned
+- The Linked Data Label and URI/L pairing selected during the LoD reconciliation process
```

Comment:
URI/L ?

Reply:
Ok, changed all of these to URL for consistency + simplicity sake